src/Orc.Theming/Models/OverrideCursorScope.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Orc.Theming;
+
+using System;
+using System.Collections.Generic;
+using System.Windows.Input;
+
+public sealed class OverrideCursorScope : IDisposable
+{
+    private static readonly Stack<Cursor> Cursors = new();
+
+    public OverrideCursorScope(Cursor changeToCursor)
+    {
+        Cursors.Push(changeToCursor);
+
+        if (Mouse.OverrideCursor != changeToCursor)
+        {
+            Mouse.OverrideCursor = changeToCursor;
+        }
+    }
+
+    public void Dispose()
+    {
+        Cursors.Pop();
+
+        var cursor = Cursors.Count > 0 ? Cursors.Peek() : null;
```

Comment:
it was old code i moved it from DG library... yes, interesting how it works before

Reply:
> Missing unit tests for OverrideCursorScope (where I think I found a bug). Please add them.

No class - no tests) - removed it, because it's redundant